infrastructure/terraform/demos/analytics/main.tf

```diff
@@ -75,7 +75,7 @@ data "template_file" "values_yaml" {
 
 resource "helm_release" "airy_core" {
   name  = "airy-release"
-  chart = "https://airy-core-helm-charts.s3.amazonaws.com/stable/airy-${replace(file("${path.module}/../../../../VERSION"), "\n", "")}.tgz"
```

Comment:
I suppose that this should be reverted before merging

Reply:
got it 👍